index.html

```diff
@@ -46,13 +47,15 @@
               <a href="#" data-activates="mobile-demo" class="button-collapse"><i class="material-icons">menu</i></a>
 
                 <ul class="tabs hide-on-med-and-down">
-                  <li class="tab col s3"><a ng-click="tc.setTab('/home')"  ng-class="{active:tc.isSet('/home')}">Home</a></li>
-                  <li class="tab col s3"><a ng-click="tc.setTab('/languages')"  ng-class="{active:tc.isSet('/languages')}">Languages</a></li>
-                  <li class="tab col s3"><a ng-click="tc.setTab('/getinvolved')"  ng-class="{active:tc.isSet('/getinvolved')}">Get Involved</a></li>
-                  <li class="tab col s3"><a ng-click="tc.setTab('/tryonline')"  ng-class="{active:tc.isSet('/tryonline')}">Try online!<span class="new badge" data-badge-caption="">beta</span></a></li>
+                  <li class="tab"><a ng-click="tc.setTab('/home')"  ng-class="{active:tc.isSet('/home')}">Home</a></li>
+                  <li class="tab"><a ng-click="tc.setTab('/about')"  ng-class="{active:tc.isSet('/about')}">About</a></li>
+                  <li class="tab"><a ng-click="tc.setTab('/languages')"  ng-class="{active:tc.isSet('/languages')}">Languages</a></li>
+                  <li class="tab"><a ng-click="tc.setTab('/getinvolved')"  ng-class="{active:tc.isSet('/getinvolved')}">Get Involved</a></li>
+                  <li class="tab"><a ng-click="tc.setTab('/tryonline')"  ng-class="{active:tc.isSet('/tryonline')}">Try online!<span class="new badge" data-badge-caption="">beta</span></a></li>
                 </ul>
                 <ul class="side-nav" id="mobile-demo">
                   <li class="tab col-md-12"><a ng-click="tc.setTab('/home')"  ng-class="{active:tc.isSet('/home')}">Home</a></li>
+                  <li class="tab col-md-12"><a ng-click="tc.setTab('/about')"  ng-class="{active:tc.isSet('/about')}">About</a></li>
```

Comment:
this is only for phones and small screens, so that it takes all 12 columns inside the side navbar.